badge/Badge/Badge.css

```diff
@@ -0,0 +1,86 @@
+@import "../../utility/component.css";
+/* / badge/ */
+
+.badge {
+    display: inline-block;
+    padding: 0.25em 0.4em;
+    font-size: 75%;
+    font-weight: 700;
+    line-height: 1;
+    text-align: center;
+    white-space: nowrap;
+    vertical-align: baseline;
+    border-radius: 0.25rem;
+}
+
+/* / Icon badge / */
+
+.badge-icon {
+    display: inline-block;
+    padding: 0.25em 0.4em;
+    font-size: 0.7rem;
+    font-weight: 700;
+    line-height: 1;
+    text-align: center;
+    white-space: nowrap;
+    vertical-align: baseline;
+    border-radius: 1rem;
+}
```

Comment:
same properties used in these 2 classes. you can make another class for same properties

Reply:
done